### ISSUE TITLE
nce_supervised_easy

### ISSUE THREAD

**Issue by topDreamer** [rjiang03/H-SCL#1]
Great work！
What is the difference between the implementation of "nce_supervised_easy" and SupCon[https://github.com/HobbitLong/SupContrast/blob/master/losses.py]?

**Comment by rjiang03** [comment]
Thank you for asking! They are the same thing, we use nce_supervised_easy as the baseline.

**Comment by topDreamer** [comment]
Thanks for the reply！
The input of SupCon is (features, labels), but the input of "nce_supervised_easy" is (out_1, out_2, label), what does "out_1, out_2" refer to? How to align with "features"?

**Comment by rjiang03** [comment]
Than you for asking!
out_1 and out_2 are the features for augmentation 1 and 2.
You can check my code in line 28. "out = torch.cat([out_1, out_2], dim=0)". We concatenate these two matrices together to get the same feature as in the SupCon.
